<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bmr</class>
 <widget class="QDialog" name="bmr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BMR</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="8" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QPushButton" name="pushButton_oblicz">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Oblicz</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Podaj płeć [k/m]:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLineEdit" name="lineEdit_wiek">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="8" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QPushButton" name="pushButton_wroc">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Wróć</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLineEdit" name="lineEdit_waga">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Podaj wiek:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLineEdit" name="lineEdit_plec">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Twoje zapotrzebowanie kaloryczne wynosi:</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLineEdit" name="lineEdit_bmr">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Podaj wagę [kg]:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Podaj wzrost [cm]:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLineEdit" name="lineEdit_wzrost">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="label_5">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>BMR</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignVCenter">
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>75</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="plainText">
          <string>BMR (z ang. basal metabolic rate), czyli wskaźnik podstawowej przemiany materii (polski skrót PPM), nazywany również tempem metabolizmu podstawowego, wskazuje najmniejsze dzienne zapotrzebowanie kaloryczne, dzięki któremu zachowamy podstawowe funkcje życiowe</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
